src/en/news/blog/2022/ceph-user-survey-results-2022/index.md

```diff
@@ -0,0 +1,110 @@
+two storage services with their ceph clusters in 2022 than 2021, but 5% fewer
+reported delivering all three. Could it be related to the decline of CephFS and
+increase over the years in RGW and market trends toward global collaboration
+systems that tend to work nicely across hybrid storage implementations and
+preferring to use S3?
+
+
+## Faster uptake for Pacific vs. Octopus
+
+When we took the survey last year, 48% had already adopted Octopus to some
+extent.  We tend to take the survey around the same time each year, and we have
+had consistently timed major releases (more or less) in the past. This year 60%
+had already adopted some form of Pacific. It’s important to note that many of
+those in both cases were running multiple versions of Ceph.
+
+
+## Where did everybody go?
+
+We had 127 responses to the user survey in 2022, which is a nearly 50% drop in
+responses year over year. We looked at changes in respondent demographics year
+over year across things like cluster size, geography, version of Ceph in use,
+and others, and we did not find significant swings that would indicate
+a particular population group that just didn’t respond this year. We have
+several hypotheses for this, and the results blog might provide an interesting
+forum for testing them out.
+
+**Hypothesis A** - it was too long, and that’s what stopped a bunch of people
+from tuning in. If you’re one of the people who didn’t respond, I’ve got
+a [super-short survey for you here](https://survey.zohopublic.com/zs/OwD7RY)
+- simple yes or no question - did you not respond because it was too long?
+
+**Hypothesis B** - you didn’t take the survey because you didn’t hear about it.
+We’ve gone hard and heavy trying to get notification of this post out to all
+the major ways that people like to get communications (based on the survey
+data) to try to reach you. Again, here’s an [extremely short
+surve](https://survey.zohopublic.com/zs/wJD7oq)y where you can tell us whether
+this contributed to your lack of response if you didn’t respond.
+
+**Hypothesis C** - low motivation. It’s possible that people didn’t respond
+because they don’t see the benefits of their perspectives reflected in the
+priorities of the community. That may be more a problem of communication than
+reality though, and so it’s important that the Ceph developers reflect back to
+their community how they prioritize their activities. Here are just a few
+examples from Josh Durgin of the way your feedback gets reflected in real
+action
+
+
+* Driven by community interaction, including data from the user survey, the
+  Ceph engineering team has prioritized  development process improvements.
+  Increased focus on reliability at scale included larger and more realistic
+  scale testing for the Quincy release. Release candidates were published for
+  users to try out and report issues before declaring a stable release.
+* The survey influenced Red Hat hiring a technical writer to focus on enhancing
```

Comment:
this should say 'the Ceph foundation' instead of Red Hat

Reply:
Another great catch!